attribute-masking-service/src/component-tests/java/uk/gov/gchq/palisade/component/attributemask/model/AttributeMaskingRequestTest.java

```diff
@@ -15,90 +15,58 @@
  */
 package uk.gov.gchq.palisade.component.attributemask.model;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
 import org.junit.jupiter.api.Test;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.autoconfigure.json.JsonTest;
-import org.springframework.boot.test.json.JacksonTester;
-import org.springframework.boot.test.json.JsonContent;
-import org.springframework.boot.test.json.ObjectContent;
-import org.springframework.test.context.ContextConfiguration;
 
-import uk.gov.gchq.palisade.Context;
-import uk.gov.gchq.palisade.User;
-import uk.gov.gchq.palisade.resource.LeafResource;
-import uk.gov.gchq.palisade.resource.impl.FileResource;
-import uk.gov.gchq.palisade.resource.impl.SystemResource;
-import uk.gov.gchq.palisade.rule.Rules;
-import uk.gov.gchq.palisade.service.SimpleConnectionDetail;
+import uk.gov.gchq.palisade.service.attributemask.common.Context;
+import uk.gov.gchq.palisade.service.attributemask.common.User;
+import uk.gov.gchq.palisade.service.attributemask.common.resource.impl.FileResource;
+import uk.gov.gchq.palisade.service.attributemask.common.resource.impl.SystemResource;
+import uk.gov.gchq.palisade.service.attributemask.common.rule.Rules;
+import uk.gov.gchq.palisade.service.attributemask.common.service.SimpleConnectionDetail;
 import uk.gov.gchq.palisade.service.attributemask.model.AttributeMaskingRequest;
 
-import java.io.IOException;
-
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.junit.jupiter.api.Assertions.assertAll;
 
-@JsonTest
-@ContextConfiguration(classes = {AttributeMaskingRequestTest.class})
 class AttributeMaskingRequestTest {
 
-    @Autowired
-    private JacksonTester<AttributeMaskingRequest> jsonTester;
-
-    @Test
-    void testContextLoads() {
-        assertThat(jsonTester).isNotNull();
-    }
-
     /**
-     * Grouped assertion test
      * Create the object with the builder and then convert to the Json equivalent.
      * Takes the JSON Object, deserialises and tests against the original Object
      *
-     * @throws IOException throws if the {@link AttributeMaskingRequest} object cannot be converted to a JsonContent.
-     *                     This equates to a failure to serialise or deserialise the string.
+     * @throws JsonProcessingException throws if the {@link AttributeMaskingRequest} object cannot be converted to a JsonContent.
```

Comment:
Statement needs rewording.  JsonProcessingException can be thrown for either the writeValueAsString or the readValue, i.e. of either an issue with serialising or deserialising.   Also, the reference to a JsonContent object is no longer valid.  This test no longer uses this Spring object.

Reply:
Minor point, the tests have been simplified down to the level that they are not grouped tests anymore and so the name of the test does not match with that it does.  Could refactor the name of tests, for example this one could be changed to testAttributeMaskingRequestSerialisingAndDeserialising.  Generalisation that could be applied to each of the test that have been refactored.

Reply:
updated